<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginMainWindow</class>
 <widget class="QMainWindow" name="loginMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
border-image: url(:/image-3/登录背景.jpg);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(240, 240, 240);
    selection-background-color: darkgray;
}</string>
    </property>
    <property name="placeholderText">
     <string>                   username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="losePwdButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>430</y>
      <width>119</width>
      <height>37</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>忘记密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enterButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,50);
 border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    selection-background-color: darkgray;</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>290</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文细黑</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: rgb(240, 240, 240);
    selection-background-color: darkgray;
}</string>
    </property>
    <property name="placeholderText">
     <string>                   password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>461</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>119</width>
      <height>37</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(85, 170, 255);</string>
    </property>
    <property name="text">
     <string>立即注册</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>段宁毛笔行书</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>扫码关注，更多详情</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
